chatbot.js

```diff
@@ -16,30 +16,30 @@ function bot() {
     setTimeout(timedQuestion, 2000);									// output next question after 2sec delay
     }
 //Questions and responses
-    else if (questionNum == 1 & input === "Yes" || input === "Sure") {
+    else if (questionNum == 1 & input === "Yes" || input === "Sure" || input === "yes" || input === "sure") {
     output.innerHTML = 'Alright. Then here you go, press the button below and look at random memes from Reddit. Enjoy! :)';
```

Comment:
You missed one ampersand:
`questionNum == 1 **&&**`
you also need to put the second condition in parentheses, and it would be better to convert all user input to lowercase by default - no matter how they type it. My suggestion:
`else if (questionNum === 1 && (input.toLowerCase() === "yes" || input.toLowerCase() === "sure"))`

Reply:
Thank you very much. Will do that :)